source-boilerplate/boilerplate.go

```diff
@@ -131,24 +134,28 @@ func (s *ConnectorServer) Capture(stream pc.Connector_CaptureServer) error {
 				return err
 			}
 		case request.Discover != nil:
+			log.WithField("eventType", "connectorStatus").Info("Discovering tables")
```

Comment:
Just as an FYI, these logs won't get plumbed through as the status because they don't actually get added to the ops logs collections. It won't hurt anything to have them here, but it might be confusing to have them. The status updates during `Open` and `Apply` would get plumbed through though.

Reply:
Yeah, that's a good point. I've removed this log message for discover RPCs.